test/integration/test-released.js

```diff
@@ -49,10 +43,6 @@ function runTest(shouldKillPolyfillableApis) {
       });
     });
 
-    // There is really weird behavior when running this test in FF in
-    // saucelabs.
-    // It never renders the ad, even though it appears to work when looking
-    // at the rendering. The test passes when running locally in FF.
```

Comment:
Don't delete this comment because it explains why skipFirefox() is set here.

Reply:
Reinstated comment to mention FF and cross-browser testing. Will try unskipping in a subsequent cleanup PR.